Packs/IvantiConnectSecure/README.md

```diff
@@ -21,7 +20,9 @@ This section describes the mandatory steps you should perform on Ivanti Connect
          | :---               | :---
          | `Server name/IP`   | Enter the [Cortex XSIAM Broker VM](#broker-vm) server's IP address or FQDN.
          | `Type`             | Select **UDP**.
-         | `Filter`           | Select **Standard** (the default).
+         | `Filter`           | Select **WELF**.
+
+**Note:** We truly recommend using the WELF log format for this pack. However, we are still supporting both log formats "WELF" and "Deafult RAW".
```

Comment:
We recommend using the WELF log format for this pack, although we still support both log formats, "WELF" and "Default RAW".